pkg/controller/keyvault/spc/nginx.go

```diff
@@ -0,0 +1,115 @@
+package spc
```

Comment:
can we rename this to nic or nginxingresscontroller?

Reply:
done